package.json

```diff
@@ -1,8 +1,19 @@
 {
   "name": "chcli",
-  "version": "1.0.0",
+  "version": "1.0.2",
   "description": "A command line utility to make create clubhouse stories easier",
-  "main": "index.js",
+  "main": "dist/src/index.js",
+  "publishConfig": {
+    "registry": "http://npm.adrise.tv"
+  },
+  "preferGlobal": true,
+  "bin": {
```

Comment:
suggestion: we should add a `files` property, most of package users don't need to install the ts files.
https://docs.npmjs.com/files/package.json#files.